hab-pkg-deb/plan.sh

```diff
@@ -0,0 +1,80 @@
+# This file is the heart of your application's habitat.
+# See full docs at https://www.habitat.sh/docs/reference/plan-syntax/
+
+pkg_name=hab-pkg-deb
+pkg_origin=yzl
+pkg_version=0.1.0
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('Apache-2.0')
+pkg_source=nosuchfile.tar.gz
+pkg_deps=(
+  core/bash
+  core/coreutils
+  core/grep
+  core/sed
+  core/util-linux
+  core/dpkg
+    core/findutils
```

Comment:
These don't need to be indented 4 spaces; only 2. The whole list should be alphabetized.